vinnie/base.py

```diff
@@ -84,7 +88,7 @@ def push(self, remote):
         if self.config.push:
             self.backend.push(remote)
         else:
-            warnings.warn("Skipping push.", UserWarning)
+            print("Skipping push.", file=sys.stderr)
```

Comment:
What's the reason behind this change? I'd rather use `warnings` for more flexibility.

Reply:
Expected user facing output shouldn't have stack traces nor should it have filenames & line numbers. This change suppresses the latter.

Reply:
We can discuss this change separately. Could you file an issue for it?

Reply:
Sure, split to #10 / #11